Source/BugsnagClient.m

```diff
@@ -625,7 +630,7 @@ - (void)addTerminationObserver:(NSString *)name {
 
 - (void)computeDidCrashLastLaunch {
     const BSG_KSCrash_State *crashState = bsg_kscrashstate_currentState();
-#if TARGET_OS_TV || TARGET_OS_IPHONE
```

Comment:
Deliberately not simulator?

Reply:
Removed change here and in `BSGOutOfMemoryWatchdog` as this appears to be deliberate